templates/includes/sidebar.html

```diff
@@ -1,9 +1,9 @@
 <ul class="sidebar-nav list-group">
   {% if user.is_authenticated %}
     <li class="list-group-item">User: <a href="{{ user.get_absolute_url }}">{{ user.get_username }}</a></li>
-    <li class="list-group-item"><a href="{% url 'logout'%}?next={{request.path}}">Logout</a></li>
+    <li class="list-group-item"><a href="{% url 'logout' %}?next={{ request.path }}">Logout</a></li>
   {% else %}
-    <li class="list-group-item"><a href="{% url 'login'%}?next={{request.path}}">Login</a></li>
+    <li class="list-group-item"><a href="{% url 'login' %}?next={{ request.path }}">Login</a></li>
   {% endif %}
 
   <br>
```

Comment:
Ensure that you use 2 spaces for indentation in your HTML files as per the checklist requirement.